lib/ansible/plugins/connection/winrm.py

```diff
@@ -47,7 +47,7 @@
     from winrm import Response
     from winrm.protocol import Protocol
 except ImportError:
-    raise AnsibleError("winrm is not installed")
+    raise AnsibleError("winrm or requests is not installed")
```

Comment:
Well, python-requests_ntlm was a correct dependency for python-winrm, but it does not pull python-requests on RHEL7/EPEL. So in my case python-requests_ntlm was already installed and not the problem. Can't talk about pip-installations though...

If you prefer requests-ntlm too, I'll add it.